starter-code/index.html

```diff
@@ -1,42 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
-<body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
 
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
+<body>
+  <nav>
+    <div class="nav-left">
+      <img src="images/spotify-logo.png" alt="spotify logo" height="60px">
+    </div>
+    <div class="nav-right">
+      <ul>
+        <li>Premium</li>
+        <li>Discover</li>
+        <li>Help</li>
+        <li>Download</li>
+      </ul>
+    </div>
+  </nav>
 
-HD Music
-Listen to music as if you
-were listening live
+  <header>
+    <div class="landing">
+      <h1>Music for everyone.</h1>
+      <p>Spotify is now free on mobile, tablet and computer. Listen to the right music, wherever you are.</p>
+    </div>
+  </header>
 
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
+  <section id="whats-on">
+    <h2>What’s on Spotify?</h2>
+    <div class="spotify-boxes">
+      <div class="box">
+        <img src="images/music-icon.png" alt="music" height="110px">
+        <h3><b>Millions of Songs</b></h3>
+        <p>There are millions of songs on Spotify</p>
+      </div>
+      <div class="box">
+        <img src="images/high-quality-icon.png" alt="sound" height="110px">
+        <h3><b>HD Music</b></h3>
+        <p>Listen to music as if you were listening live</p>
+      </div>
+      <div class="box">
+        <img src="images/devices-icon.png" alt="device" height="110px">
+        <h3><b>Stream Everywhere</b></h3>
+        <p>Stream music on your smartphone, tablet or computer</p>
+      </div>
+    </div>
+  </section>
 
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <section id="green">
+    <div class="green-background">
+      <div class="text">
+        <h2>It’s as yeezy as Kanye West.</h2>
+        <div class="text-under">
+          <h3>Search</h3>
+          <p>Know what you want to listen to? Just search and hit play.</p>
+          <h3>Browse</h3>
+          <p>Check out the latest charts, brand new releases and great playlists for right now.</p>
+          <h3>Discover</h3>
+          <p>Enjoy new music every Monday with your own personal playlist. Or sit back and enjoy Radio.</p>
+        </div>
+      </div>
+      <div class="small-logo"><img src="images/spotify-icon-white.png" alt="white logo" width="100px"></div>
+      <div class="player"><img src="images/spotify-app.jpg" alt="spotify app" width="80%"></div>
```

Comment:
Setting a relative width to your items can be problematic, because different screen sizes will have a greater impact on your styling. Since you are using flexbox, try to take advantage of the features that allow for items to be distributed dynamically inside their flex-containers.